network-api/networkapi/buyersguide/templates/about/how_to_use.html

```diff
@@ -13,232 +13,319 @@ <h1 class="h2-heading">
+      the same company. We simply don’t have time to review all the connected products out there.
       {% endblocktrans %}
     </p>
   </section>
 
-  <section aria-labelledby="encryption">
-    <p id="encryption"><b>{% trans "Encryption" %}</b></p>
+  <section aria-labelledby="warning-label">
+    <p id="warning-label"><strong>{% trans "*Privacy Not Included Warning Label" %}</strong></p>
     <p class="body">
       {% blocktrans trimmed %}
-      Encryption is your friend. It protects your private and personal information by scrambling it up into a code so
-      that the only people or machines who can read it are the ones on the other end who have the key to unscramble that
-      code. Products that don’t use encryption send personal information over the internet unscrambled so anyone can see
-      it.
+      Our <em>*Privacy Not Included</em> buyer’s guide now comes with <em>*Privacy Not Included</em>
+      warning labels on products we think consumers should think twice about before buying. It’s no
+      small thing to assign such a label to a product, so we set ourselves some strict standards.
+      If we can’t confirm a product meets our Minimum Security Standards, it automatically earned
+      the <em>*Privacy Not Included label</em>, as we feel those standards are the minimum a product
+      should meet to be on the market. We also look at how a company uses the data the product
+      collects on you, how you can control the data the company collects, and what the company’s
+      known track record is over the past two years for protecting their users’ data. How a company
+      performed on these criteria determined if we assigned them the warning label. You will see
+      little mini-warnings in our review section of the guide to help you understand what our concerns are.
       {% endblocktrans %}
     </p>
   </section>
 
-  <section aria-labelledby="security-updates">
-    <p id="security-updates"><b>{% trans "Security updates" %}</b></p>
+  <section aria-labelledby="what-could-go-wrong">
+    <p id="what-could-go-wrong"><strong>{% trans "What could happen if something goes wrong" %}</strong></p>
     <p class="body">
       {% blocktrans trimmed %}
-      Sometimes security vulnerabilities are found in products after they are sold to the public. For that reason,
-      companies should have a way to quickly push a security update out to the product automatically so it fixes the
-      security vulnerability without the consumer ever needing to worry about it.
+      It’s likely nothing bad will happen with most of the products in this guide. However, it’s
+      also good to think through what could happen if something goes wrong. We explain what
+      concerns users should have about the product and often lay out a potential worst-case
+      scenario&mdash;in some cases for fun and in some cases based on things that have already happened.
```

Comment:
we use ` &mdashl ` elsewhere, so we probably want to be consistent in either adding spaces, or not adding spaces around em dashes.

Reply:
@TheoChevalier This affects your work mostly. Thoughts on spaces around the mdash?